cmd/cli/plugin/package/main.go

```diff
@@ -76,3 +77,15 @@ func isPackagingAPIAvailable(kubeCfgPath string) (bool, error) {
 
 	return clusterQueryClient.Query(apiGroup1, apiGroup2, apiGroup3, apiGroup4).Execute()
 }
+
+func getDefaultNameSpace() string {
+	loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
+	loadingRules.ExplicitPath = kubeConfig
+	configOverrides := &clientcmd.ConfigOverrides{}
+	kubeConfig := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(loadingRules, configOverrides)
+	namespace, _, err := kubeConfig.Namespace()
+	if err != nil || namespace == "" {
```

Comment:
question: Do we want to some logging here in the case of error? And a little bit curious on what is the user experience if `kubeconfig` provided does not exist(`loadingRules.ExplicitPath = kubeConfig`).

Reply:
^^ yes. The error does not require process to stop but the user should be informed because it means there might be something bad with their kubeconfig

Reply:
Added error log pointing out failue to obtain a valid context namespace from the available kubeconfigs.